src/cpp/session/modules/build/SessionBuild.cpp

```diff
@@ -2343,6 +2343,7 @@ Error initialize()
       (bind(registerRpcMethod, "devtools_load_all_path", devtoolsLoadAllPath))
       (bind(registerRpcMethod, "get_bookdown_formats", getBookdownFormats))
       (bind(sourceModuleRFile, "SessionBuild.R"))
+      (bind(sourceModuleRFile, "SessionInstallRtools.R"))
```

Comment:
Does this need to be "build/SessionInstallRtools.R"? (Note that `SessionBuild.R` is actually located in the modules directory, not the modules/build directory -- I'm not sure if that's an oversight, though)

Reply:
I tried it different ways, but this way works. I was confused because I noticed that about SessionBuild.R code path. However, since I can find `.rs.findRtools42Installer()` in the R session it must be sourcing the file correctly

Reply:
(I don't really get how these paths work, but trial and error led me to this solution)

Reply:
It's possible that sub-directories aren't respected when module files are copied to the build tree, so perhaps this is fine. If it works, then we can leave it :-)

Reply:
Yep, in the build directory, all of these files are under `session/modules/R`